Chrome user device characteristics report (#1440)
GoogleChrome/developer.chrome.com: site/en/blog/device-characteristics/index.md

---
layout: 'layouts/blog-post.njk'
title: Chrome user device characteristics report
description: >
  Developers constantly wonder what kind of audience they are developing for. How much RAM do they have? What kind of Wi-fi are they on? A new report provides answers for a range of countries and platforms.
date: 2021-09-22
authors:
  - jichen
tags:
  - device-characteristics
hero: 'image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/SJJ2a5QPJ2jSsNyGDbTh.jpeg'
alt: >

>   An alternative text description of your hero or social image.

Reviewer: This needs to be filled out

---

Developers are constantly wondering about what kind of audience they are
developing for. Are they using high end devices with large RAM? What connection
are they using (for example Wi-Fi, 2G) and so on. The answers to these questions
can affect product development decisions such as choosing the
technologies to render the content. This report provides timeline data to inform
developers on these questions for a list of countries, such as the US, Brazil,
Japan, and others. Three platforms are included when data is available: Android,
Windows, and ChromeOS.

Looking at the data, there might be some interesting things for developers to
think about. For example, in Japan on the Android platform, the majority of
users have five to eight cores, and the prevalence is quite a bit higher than in
the US, and much higher than in next billion user (NBU) countries. Another
example: if you look at RAM on Android, a very low percentage of Brazil devices

> have RAM larger than 5GB, which is much lower than in the US, and also lower

Reviewer 2: ```suggestion
have RAM larger than 5&nbsp;GB, which is much lower than in the US, and also lower
```

than India. We hope publishing information like this periodically will help
developers have a better understanding of the Chrome user device
characteristics.

This data comes from usage statistics sent to Google from Chrome installations.
In Chrome, users can choose to send usage statistics and crash reports to Google
to help improve Chrome's feature set and stability. Usage statistics and crash
reports are enabled by default for Chrome consumer installs and can be disabled
during installation or in Chrome's settings. Metrics collection is pseudonymous,
and metrics are tagged by an opaque per-install identifier that is not joinable
with other datasets.  If the metrics collection is re-enabled, the unique
identifier will also be reset. We used the data collected to calculate the
various statistics in this

> [report](https://docs.google.com/document/d/e/2PACX-1vTLB93N1X3QRs_0T869wIppeXYDGWeyDeVM5N8XSKVAx3cIl1RyNj3fM4gUn06cEimfUM4wubmC_laB/pub).

Reviewer: I cannot access this document in Incognito or via my personal account.

> Photo by [Halacious](https://unsplash.com/@halacious?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText) on [Upsplash](https://unsplash.com/s/photos/multiple-phones?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText)

Reviewer: nb: Upsplash should be Unsplash
Also, remove the `utm_source` and friends.

---
layout: 'layouts/blog-post.njk'
title: Chrome user device characteristics report
description: >
  Developers constantly wonder what kind of audience they are developing for. How much RAM do they have? What kind of Wi-fi are they on? A new report provides answers for a range of countries and platforms.
date: 2021-09-17
authors:
  - jichen
tags:
  - device-characteristics
hero: 'image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/zIWz3bDI3mBRLki1VxMJ.jpeg'
alt: >
  An alternative text description of your hero or social image.
---

Developers are constantly wondering about what kind of audience they are
developing for. Are they using high end devices with large RAM? What connection

> are they using (wifi, 2G...)? etc. The answers to such questions could carry

Reviewer 2: ```suggestion
are they using (for example Wi-Fi, 2G) and so on. The answers to such questions could carry
```

> much weight in many aspects of product development such as choosing the right

Reviewer 2: ```suggestion
can affect product development decisions such as choosing the
```

technologies to render the content. This report provides timeline data to inform

> developers on these questions for a list of countries, like the US, Brazil,

Reviewer 2: ```suggestion
developers on these questions for a list of countries, such as the US, Brazil,
```

> Japan, etc. Three platforms are included when data is available: Android,

Reviewer 2: ```suggestion
Japan, and others. Three platforms are included when data is available: Android,
```

Windows, and ChromeOS.

Looking at the data, there might be some interesting things for developers to
think about. For example, in Japan on the Android platform, the majority of

> users have five to eight cores, and the prevalence is quite a bit higher than

Reviewer 2: ```suggestion
users have five to eight cores, and the prevalence is quite a bit higher than in
```

> the US, and much higher than in NBU countries. Another example: if you look at

Reviewer 2: ```suggestion
the US, and much higher than in NBU (next billion users) countries. Another example: if you look at
```

the smallest display size in device pixels, in some countries like the US or

> Brazil, there's a clear trend that smallest display sizes (like 400-500) are

Reviewer 2: This seems like a weird way to refer to display size. 400px in which dimension, x, y, diagonal? Looking at this [chart of iPhone resolutions](https://www.ios-resolution.com/), some legitimately big phones such as 12 pro max are 428 x 926. (The chart says that's logical pixels, but I think it's mislabeled.) Also, consider adding a link to an explainer of device pixels vs logical pixels.

becoming more and more popular, while in some other countries like India, the
mix looks pretty stable and there does not seem to be a trend. We hope
publishing information like this periodically will help developers have a better
understanding of the Chrome user device characteristics.

This data comes from usage statistics sent to Google from Chrome installations.
In Chrome, users can choose to send usage statistics and crash reports to Google
to help improve Chrome's feature set and stability. Usage statistics and crash
reports are enabled by default for Chrome consumer installs and can be disabled

> during installation or in Chrome's settings. Metrics collecting is pseudonymous,

Reviewer 2: ```suggestion
during installation or in Chrome's settings. Metrics collection is pseudonymous,
```
